_posts/2022-10-13-opening-reception.md

```diff
@@ -0,0 +1,30 @@
+---
+author: Lacey Williams Henschel
+category: General
+date: 2022-10-13 06:00:00
+layout: post
+image: /static/img/blog/opening_reception.png
+title: "Opening Reception at DjangoCon US"
+---
+
+We're so excited to tell you about your opportunities to socialize with and get to know other attendees, speakers, organizers, and sponsors!
+
+## Opening Reception
+
+**When**: 5:45-7:00 pm, Monday, September 23
+
+**Where**: West Lawn at the Marriott (on the other side of the pool from the Marriott restaurant)
+
+**What**: Join us for some food and non-alcoholic drinks to relax after the first day of talks! We'll have light appetizers for you to enjoy.
+
+Also don't forget our other social event, board game night!
+
+## Board Game Night
```

Comment:
Don't we already have this in the other post?

Reply:
Yep, I was rehashing it to have an all in one page for Carol to tweet about.